web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -667,6 +667,9 @@ const UpstreamForm: React.FC<Props> = forwardRef(
                   name: formatMessage({ id: 'page.upstream.step.select.upstream.select.option' }),
                   id: '',
                 },
+                {
```

Comment:
Hi @stu01509 
I noticed that we already have `None` option, see
https://github.com/apache/apisix-dashboard/pull/1633/files#diff-0029901a283985aea1e8e47cfb2d45c9c8eef8cb392e12ed31720390a22ec269R664

and this will be controlled by `required` param, you can try to pass `required={false}` in 
https://github.com/apache/apisix-dashboard/blob/9c27b57dc5d2549706a8348f1a036743246f0337/web/src/pages/Service/components/Step1.tsx#L57

Reply:
aha! Good catch!

Reply:
Hi @liuxiran 

I'm not sure your mean, and I didn't found the `None` option in the current codebase,
could you guide me how to approach it?

Thanks

Reply:
Hi @stu01509 
1. turn to this line https://github.com/apache/apisix-dashboard/pull/1633/files#diff-0029901a283985aea1e8e47cfb2d45c9c8eef8cb392e12ed31720390a22ec269R664, you will see 
![image](https://user-images.githubusercontent.com/2561857/112560691-b0184900-8e0e-11eb-8e7e-8991ceb3761f.png)

this is the `None` option item, and you will find that it is controlled by `required` parameter
3. for the `upstreamForm` component, `required` is a props item, see
![image](https://user-images.githubusercontent.com/2561857/112560919-1f8e3880-8e0f-11eb-923b-a23aa5b72d60.png)

4. our create service page used `upstreamForm` component, see
![image](https://user-images.githubusercontent.com/2561857/112561354-ff12ae00-8e0f-11eb-9295-595ddc8ce3d2.png)

you can try to change the `required` param to `required={false}`, that will solve your problem.

just try it ^_^, looking forward to your reply, thanks